uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/LeavePeriodEntitlement.php

```diff
@@ -299,14 +299,14 @@ private static function savePublicHolidaysBalanceChanges(
   ) {
     $balanceChangeTypes = array_flip(LeaveBalanceChange::buildOptions('type_id', 'validate'));
 
-    $publicHolidays = $calculation->getPublicHolidaysInEntitlement();
+    $publicHolidays = $calculation->getNumberOfPublicHolidaysInEntitlement();
```

Comment:
The name of this variable should reflect that it stores the number of public holidays and not just public holidays.